plugins/field-colour/test/index.ts

```diff
@@ -9,38 +9,261 @@
  */
 
 import * as Blockly from 'blockly';
-import {generateFieldTestBlocks, createPlayground} from '@blockly/dev-tools';
-import '../src/index';
+import {javascriptGenerator} from 'blockly/javascript';
+import {dartGenerator} from 'blockly/dart';
+import {phpGenerator} from 'blockly/php';
+import {pythonGenerator} from 'blockly/python';
+import {luaGenerator} from 'blockly/lua';
 
-const toolbox = generateFieldTestBlocks('field_colour', [
+import {createPlayground} from '@blockly/dev-tools';
+import {installAllBlocks as installColourBlocks} from '../src/index';
```

Comment:
I'm not sure this rename is useful (unlike the similar one in the readme, which I think is useful.)

Reply:
I found it useful since I have several sets of blocks that I'm working with here. Do you have a strong opinion about it?

Reply:
No, it is not objectionable.